vignettes/b-advanced-ggplot.Rmd

```diff
@@ -67,7 +67,20 @@ ggplot(penguins_subset) +
   theme_bw(base_size = 15)
 ```
 
-That's a nice basic plot! But you know what would make it nicer? If we used little penguins instead of points! To do that, we can use the `geom_phylopic()` function instead of the `geom_point()` function. Note that we need to wrap the penguin image in a `list()` call to make it a repeatable object (in this case, we want to use the same image for each x-y pair):
+That's a nice basic plot! But you know what would make it nicer? If we added a penguin silhouette to the plot. Sadly, we don't have a different silhouette for each species (although we could make one...), so let's just go with putting a single silhouette in the top panel. We'll use the `geom_phylopic()` function, which will require us to make a data frame. Note that the `x` and `y` aesthetics specify the center of the silhouette, and the `size` argument specifies how tall the silhouette is in the units of the y-axis. Also note that we need to wrap the penguin image in a `list()` call to make it a repeatable object
+
+```{r}
+silhouette_df <- data.frame(x = 59, y = 215, species = "Adelie")
+ggplot(penguins_subset) +
+  geom_point(aes(x = bill_length_mm, y = flipper_length_mm)) +
+  geom_phylopic(data = silhouette_df, aes(x = x, y = y), size = 30,
+                img = list(penguin_rot)) +
```

Comment:
This should work without having to wrap it in list now?

Reply:
Well...once that PR is merged it will...

Reply:
Just updated this in the wrapper PR...